src/Form/AdminSettingsForm.php

```diff
@@ -206,9 +227,13 @@ private function buildFieldMappingsForm() {
     $field_mappings_settings = $this->config('acquia_lift.settings')->get('field_mappings');
     $field_names = $this->getTaxonomyTermFieldNames();
 
+    $taxonomy_vocabulary_url_text = t('Taxonomy Vocabulary');
+    $taxonomy_vocabulary_url = Url::fromRoute('entity.taxonomy_vocabulary.collection', [], ['attributes' => ['target' => '_blank']]);
+    $taxonomy_vocabulary_link = Link::fromTextAndUrl($taxonomy_vocabulary_url_text, $taxonomy_vocabulary_url)->toString();
+
     $form = [
       '#title' => t('Field Mappings'),
-      '#description' => t('Map Taxonomy vocabulary fields to "content section", "content keywords", and "persona" fields.'),
+      '#description' => t('Create ') . $taxonomy_vocabulary_link . t(' and map to "content section", "content keywords", and "persona" fields.'),
```

Comment:
I have actually started with trying to use placeholders, and spend stupid amount of time trying to get it to work. The problem here is that I would like to keep using the Link::fromTextAndUrl() which generates an a tag, and using placeholder will escape the a tag. I went looking for examples and found that Drupal core code hardcode a tag in strings just to use the placeholders.

Anyway, this is intentional. I can give it another try, but I am not sure if it is worth the effort. Do you mind we leave it as-is?